ldm/simplet2i.py

```diff
@@ -299,11 +305,13 @@ def _txt2img(self,
                  batch_size,
                  steps,cfg_scale,ddim_eta,
                  skip_normalize,
+                 gfpgan_strength,
```

Comment:
Nit: as a matter of style, rather than threading this through into `_txt2img` and `_img2img`, it would make more sense for `prompt2image` to be responsible for calling this, by putting the call just before [this line](https://github.com/lstein/stable-diffusion/blob/cb86b9ae6e849cbe4609a62eaf686388eda9b2df/ldm/simplet2i.py#L285). That way the underlying `_txt2img` and `_img2img` can continue to only have a single responsibility.

Reply:
Yeah, that makes sense, I had already pulled when main was updated, so will go back.

Reply:
well actually, the _samples_to_images generator is being run in each of those two methods which means they still have the side effect of generating the image and writing it to disk - which means we have to pass the information down.

If we have both methods yield 'samples' enums we can then do it at the top level

Reply:
oh I see. disregard!

Reply:
Resolved, thank you!